### DISCUSSION TITLE
Suggest Terminal Command + Explain This results in error

### DISCUSSION THREAD

**Discussion by aminnaggar** [community/community#75837] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Had been working for months up until 4 or so hours ago. Now I'm getting errors. like "command 'github.copilot.terminal.suggestCommand' not found"

### GitHub Copilot
Version: `1.137.0`

### OS
I'm on Windows 11 running Ubuntu 20.04 in WSL2

### Vscode
Version: 1.84.2 (system setup)
Commit: 1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
Date: 2023-11-09T10:51:52.184Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631

## Remedies tried
* Restarting Windows
* Uninstalling and reinstalling Github Copilot
* Signing out of my Github account on Vscode resigning in
* Downgrading to `1.136.0`, also tried `1.133.0`

### Suggest Terminal Command
Palette Command: GitHub Copilot: Suggest Terminal Command

Command 'GitHub Copilot: Suggest Terminal Command' resulted in an error
command 'github.copilot.terminal.suggestCommand' not found

![image](https://github.com/community/community/assets/857186/4a950f4b-44aa-4c89-bc37-f1320e14716e)

### Explain This (Terminal)
Palette Command: GitHub Copilot: Explain This (Terminal)

Command 'GitHub Copilot: Explain This (Terminal)' resulted in an error
command 'github.copilot.terminal.explainTerminalSelection' not found

![image](https://github.com/community/community/assets/857186/aa9ccdc0-0124-4b1c-b381-ffce1acbacf0)

### Quick Chat
Palette Command: Chat: Open Quick Chat (GitHub Copilot)

Silently fails

## Other Info

No errors appear in the GitHub Copilot Output logs

### Diagnostics
Running "Show Diagnostics" spits this out

**Copilot**

- Version: 1.137.0
- Build: prod
- Editor: vscode/1.84.2

**Environment**

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

**Feature Flags**

- Custom Certificates: enabled
- Send Restricted Telemetry: disabled

**Node setup**

- Number of root certificates: 140
- Operating system: Linux
- Operating system version: 5.15.133.1-microsoft-standard-WSL2
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

**Network Configuration**

- Proxy host: n/a
- Proxy port: n/a
- Proxy auth: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled

**Reachability**

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

**VS Code Configuration**

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

**Extensions**

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

**Comment by eyalatox** [comment]
+1

**Reply by omerz-wix** [reply]
+1

**Comment by aminnaggar** [comment]
I moved off of my personal github copilot and into my work's team account. What I didn't realize is that my team had that feature turned off from github (didn't realize that it's a separate feature to begin with tbh).

Once we turned it on from the github team settings, the error went away.
I think better language here would have saved me a good few hours, maybe something like "Check your Suggestion settings in Github" would have been fantastic
